okhttp/src/main/kotlin/okhttp3/internal/http2/Http2Reader.kt

```diff
@@ -146,9 +146,15 @@ class Http2Reader(
     headerBlockLength = lengthWithoutPadding(headerBlockLength, flags, padding)
     val headerBlock = readHeaderBlock(headerBlockLength, padding, flags, streamId)
 
-    handler.headers(endStream, streamId, -1, headerBlock)
+    // Ignore early hints
```

Comment:
I would have expected this in `CallServerInterceptor`. Any benefit to doing it here?